src/app/containers/App/App.jsx

```diff
@@ -3,7 +3,7 @@
  * © Jordan Tart https://github.com/jtart
  * https://github.com/jtart/react-universal-app
  */
-import { Component } from 'react';
+import { Component, useEffect, useRef, useState } from 'react';
```

Comment:
Oops ignore this for now, this will be removed, but was refactoring the main App component to use Hooks.